rtloader/include/datadog_agent_rtloader.h

```diff
@@ -28,17 +28,6 @@ typedef struct rtloader_s rtloader_t;
 struct rtloader_pyobject_s;
 typedef struct rtloader_pyobject_s rtloader_pyobject_t;
 
-// FACTORIES
```

Comment:
Would it make sense to keep this line? I think it applies to both `make2` and `make3`.